### PR TITLE
Moved GE test cases to stable.

### PR DESCRIPTION
**Pull request by deepmalamehta**
# Critical Changes

# Changes 
These test cases were passing for continuous runs. https://salesforce.quip.com/SMoaAZvY9tHD is the doc with screenshots.

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: robot/Cumulus/tests/browser/gift_entry/error_validation_rename_fields.robot [hunk 1]
```diff
@@ -36,7 +36,7 @@ Validate Errors When Field Is Renamed
     ...                     status is dry run error. Rename the field to correct it and verify no errors or warnings are thrown on AM, object group,
     ...                     and template builder. Creates a batch and verifys field is back on form and on saving gift its status is dry run validated.
 
-    [tags]                              unstable                      feature:GE        W-8292840
```
Comment (schlossmj) [inline robot/Cumulus/tests/browser/gift_entry/error_validation_rename_fields.robot:39]: Let's hold off on marking this one stable for now - this unfortunately failed again yesterday with the component error.

Comment (deepmalamehta) [reply robot/Cumulus/tests/browser/gift_entry/error_validation_rename_fields.robot:39]: Sure

**Review by rjanjanam — COMMENTED** [review]
Are they consistently passing on both Reg and qa org flows? DId you have to do anything else other than remove the unstable tag to make them stable ?

File: src/package.xml [hunk 2]
```diff
@@ -4397,4 +4397,4 @@
         <name>WorkflowRule</name>
     </types>
     <version>48.0</version>
-</Package>
+</Package>
```
Comment (skristem) [inline src/package.xml:4400]: looks like u r accidentally commiting a change to package.xml file. You might want to discard it.

Comment (deepmalamehta) [reply src/package.xml:4400]: Thanks. I will discard it.

**Comment by deepmalamehta** [comment]
> Are they consistently passing on both Reg and qa org flows? DId you have to do anything else other than remove the unstable tag to make them stable ?

Feature test- robot uses qa_org flow and hence based this PR on that. No, I just had to remove the unstable tag.

**Review by rjanjanam — APPROVED** [review]
Looks good.

**Review by schlossmj — APPROVED** [review]

**Pull request merged**